andela/strikers-ah-frontend
# #164489767 home page

#### What does this PR do?
This PR shows the **landing page** and helps a user to navigate the website.
#### Description of Task to be completed?
- Created components necessary to display the home page.
- Added redux to handle state and data request from the server.
#### How should this be manually tested?
- Clone this repo from https://github.com/andela/strikers-ah-frontend.git 
- After, check out to develop branch by typing `git checkout develop` in your terminal.
- To start the server type `npm start` in your terminal
#### What are the relevant pivotal tracker stories?
164489767
#### Screenshots (if appropriate).
<img width="1440" alt="Screen Shot 2019-06-07 at 13 38 55" src="https://user-images.githubusercontent.com/37753967/59101663-f8a8fe80-8929-11e9-81dc-63cb47cff2f0.png">
<img width="1440" alt="Screen Shot 2019-06-07 at 13 39 11" src="https://user-images.githubusercontent.com/37753967/59101687-11b1af80-892a-11e9-85ab-fc68230a396e.png">
